lib/ace/edit_session/bracket_match.js

```diff
@@ -146,7 +207,7 @@ function BracketMatch() {
             // Scan backward through the document, looking for the next token
             // whose type matches typeRe
             do {
-                token = iterator.stepBackward();
+                token = iterator.stepBackward();              
```

Comment:
Normally it is not okay to add random whitespace changes (unless you change formatting in the whole file).